src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**Network**
+
+* Fixed issue #9861. Remove `--ids` from the parameters.
```

Comment:
The network section should be located alphabetically. Also, the typical format would be:

* `network nic ip-config add`: Fixed issue where `--ids` was inadvertently exposed but did not work. 

Also, were it not for the fact that `--ids` caused a crash, we could not simply remove this. We would need to deprecate it using: https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments

Reply:
Actually, I don't understand what is the `--ids`. Is it by design to show the `--ids` when you add `id_parts=xxx` into the `c.arguments`? What do we need as the value for this field?
I will fix other comments, thanks very much!~

Reply:
It is discussed here:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-the-ids-parameter

Reply:
Yes. I checked that doc before.  Maybe I misunderstand something, please correct me. To be simple, what should we used as an input for the `--ids` in the example below? `/subscription/xxxx-xxxx-xxxx/resoureceGroup/myResourceGroup/Microsoft.NIC/myNIC` or `/subscription/xxxx-xxxx-xxxx/resoureceGroup/myResourceGroup/Microsoft.NIC/myNIC/publicIP/mypublicIP` or both of them?
```
Resource Id Arguments
    --ids              : One or more resource IDs. If provided, no other 'Resource Id' arguments
                         should be specified.
    --name -n          : The IP config name.
    --nic-name         : The NIC name.
    --resource-group -g: Name of resource group.
```

Reply:
Neither, but that is why `--ids` was being removed from the command. `--ids` should always refer to whatever the argument group's central item is.  In this case it is questionable or confusing what `--ids` should refer to, which is why it shouldn't be exposed.